README.md

```diff
@@ -131,7 +123,7 @@ not available
 
 > Describe 1 - 3 features you will add to the project.
 
-- [ ] **More data of the animals**
+- [ ] **Add more classes**
 
 <p align="right">(<a href="#readme-top">back to top</a>)</p>
 
```

Comment:
- [ ] As requested by the previous reviewer, update the MIT license because right now, it is still giving error 404. We have to avoid having broken links in our ReadMe file to keep up with professionalism
![Screen Shot 2023-02-09 at 7 21 15 AM](https://user-images.githubusercontent.com/62849872/217855082-4b807024-2fbc-43de-a1e8-0befe2a8e8cd.png)